model_configs/dna_tuning_config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "population_size": 2,
+    "n_generations": 10,
+    "output_path": "./tuning_results",
+    "attributes": {
+        "hidden_layer_size": {
+            "int": [1, 512, 1]
```

Comment:
this structure is not readily understood. it looks like the type is int, the min is 1, max is 512, and step size/precision is 1. could we have it look like:
```
"hidden_layer_size": {
    "type": "int",
    "min": 1,
    "max": 512,
    "precision": 1
}
```
I am not sure what to call the last argument.

Reply:
Yes, this would be an excellent change. It would involve updating the `tuningdeap` package though.  It might be better to just add this as an issue in the other repo and work on it there.

Reply:
https://github.com/orionw/tuningDEAP/issues/1

Reply:
That issue is closed, and this file is updated in this repo (along with the requirements.txt)